libcloud/security.py

```diff
@@ -61,6 +65,19 @@
     '/etc/ssl/certs/YaST-CA.pem',
 ]
 
+# Insert certifi CA bundle path to the front of Libcloud CA bundle search
+# path if certifi is available
+try:
+    import certifi
+except ImportError:
+    has_certifi = False
+else:
+    has_certifi = True
+
+if has_certifi and USE_CERTIFI:
+    certifi_ca_bundle_path = certifi.where()
+    CA_CERTS_PATH.insert(0, certifi_ca_bundle_path)
```

Comment:
Sorry, this was more meant to be a question and not a suggestion.

> The thing is that `USE_CERTIFI` actually defaults to True right now. So in this case issuing a warning would be confusing.

Good point.

> I'm personally not totally opinionated on this one (yet) and I could be convinced to default it to False and issue a warning if set to True.

Personally, I prefer the current behavior (for somewhat selfish reasons). Basically, `conda` has activate scripts that one can use to set environment variables, but they are a bit flaky. There has been a lot of work recently to improve them and I do expect they will get better. Unfortunately, they just are not there yet.

While I can understand the need for explicit over implicit, `certifi` ends up being a simple and reliable way to provide what you are looking for here IIUC. Even `setuptools` provides it as an [option]( https://github.com/pypa/setuptools/blob/a73ca4fdcd1ba386159eb6ceb66d40af674a034c/setup.py#L147 ). So, I think trying to use that first by default and searching elsewhere is completely reasonable.